### ISSUE TITLE
ModuleNotFoundError: No module named 'pubgis.minimap_iterators'

### ISSUE THREAD

**Issue by dabroz** [andrewzwicky/PUBGIS#24]
PUBGIS doesn't work, after installing on OSX with `pip3.6 install pubgis` (with dependencies as stated in #23) it crashes like this:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/pubgis/__main__.py", line 7, in <module>
    from .gui import PUBGISMainWindow
  File "/usr/local/lib/python3.6/site-packages/pubgis/gui.py", line 13, in <module>
    from pubgis.minimap_iterators.generic import ResolutionNotSupportedException
ModuleNotFoundError: No module named 'pubgis.minimap_iterators'
```

A quick search in `/usr/local/lib/python3.6/site-packages/pubgis` finds no `pubgis.minimap_iterators` or `ResolutionNotSupportedException` definitions.

**Comment by andrewzwicky** [comment]
Looks like an issue with `setup.py` not including subdirectories.  I'll look into it, thanks for the report!

**Comment by andrewzwicky** [comment]
fixed in b5487144fee9d826e7dfaddc7fe51aae2b6c76ff

**Comment by andrewzwicky** [comment]
output from appveyor

```
%PYTHON%\python setup.py sdist
running sdist
running egg_info
writing PUBGIS.egg-info\PKG-INFO
writing dependency_links to PUBGIS.egg-info\dependency_links.txt
writing top-level names to PUBGIS.egg-info\top_level.txt
reading manifest file 'PUBGIS.egg-info\SOURCES.txt'
writing manifest file 'PUBGIS.egg-info\SOURCES.txt'
warning: sdist: standard file not found: should have one of README, README.rst, README.txt
running check
creating PUBGIS-0.1.4
creating PUBGIS-0.1.4\PUBGIS.egg-info
creating PUBGIS-0.1.4\pubgis
creating PUBGIS-0.1.4\pubgis\images
creating PUBGIS-0.1.4\pubgis\minimap_iterators
copying files to PUBGIS-0.1.4...
copying setup.cfg -> PUBGIS-0.1.4
copying setup.py -> PUBGIS-0.1.4
copying PUBGIS.egg-info\PKG-INFO -> PUBGIS-0.1.4\PUBGIS.egg-info
copying PUBGIS.egg-info\SOURCES.txt -> PUBGIS-0.1.4\PUBGIS.egg-info
copying PUBGIS.egg-info\dependency_links.txt -> PUBGIS-0.1.4\PUBGIS.egg-info
copying PUBGIS.egg-info\top_level.txt -> PUBGIS-0.1.4\PUBGIS.egg-info
copying pubgis\__init__.py -> PUBGIS-0.1.4\pubgis
copying pubgis\__main__.py -> PUBGIS-0.1.4\pubgis
copying pubgis\color.py -> PUBGIS-0.1.4\pubgis
copying pubgis\gui.py -> PUBGIS-0.1.4\pubgis
copying pubgis\match.py -> PUBGIS-0.1.4\pubgis
copying pubgis\match_result.py -> PUBGIS-0.1.4\pubgis
copying pubgis\images\full_map.jpg -> PUBGIS-0.1.4\pubgis\images
copying pubgis\images\full_map_scaled.jpg -> PUBGIS-0.1.4\pubgis\images
copying pubgis\images\indicator_area_mask.jpg -> PUBGIS-0.1.4\pubgis\images
copying pubgis\images\indicator_mask.jpg -> PUBGIS-0.1.4\pubgis\images
copying pubgis\minimap_iterators\__init__.py -> PUBGIS-0.1.4\pubgis\minimap_iterators
copying pubgis\minimap_iterators\generic.py -> PUBGIS-0.1.4\pubgis\minimap_iterators
copying pubgis\minimap_iterators\images.py -> PUBGIS-0.1.4\pubgis\minimap_iterators
copying pubgis\minimap_iterators\live.py -> PUBGIS-0.1.4\pubgis\minimap_iterators
copying pubgis\minimap_iterators\video.py -> PUBGIS-0.1.4\pubgis\minimap_iterators
Writing PUBGIS-0.1.4\setup.cfg
Creating tar archive
removing 'PUBGIS-0.1.4' (and everything under it)
```
